From 09c1ad8068355d3de74cf5b3263f24d4a2b5ebc2 Mon Sep 17 00:00:00 2001
From: John Johansen <john.johansen@canonical.com>
Date: Fri, 18 Mar 2016 06:06:27 -0700
Subject: [PATCH 1635/1767] Revert "UBUNTU: apparmor -- follow change to
 this_cpu_ptr"

BugLink: http://bugs.launchpad.net/bugs/1379535

This reverts commit 53f139d625d03defdc30acdac4b02ab598ff9a5f.

Signed-off-by: Tim Gardner <tim.gardner@canonical.com>
---
 security/apparmor/include/path.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/apparmor/include/path.h b/security/apparmor/include/path.h
index 79c48d4..09b47b8 100644
--- a/security/apparmor/include/path.h
+++ b/security/apparmor/include/path.h
@@ -64,7 +64,7 @@ DECLARE_PER_CPU(struct aa_buffers, aa_buffers);
 #define __get_buffer(N) ({					\
 	struct aa_buffers *__cpu_var; \
 	AA_BUG_PREEMPT_ENABLED("__get_buffer without preempt disabled");  \
-	__cpu_var = this_cpu_ptr(&aa_buffers);			\
+	__cpu_var = &__get_cpu_var(aa_buffers);			\
         __cpu_var->buf[(N)]; })
 
 #define __get_buffers(X...)		\
-- 
1.9.3

